Makefile

```diff
@@ -35,3 +55,21 @@ unit-test:
 
 build: $(SOURCES)
 	go build $(GOBUILDFLAGS) -ldflags="$(GOLDFLAGS)" -o ./update-service-operator ./
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests:
+	controller-gen $(CRD_OPTIONS) rbac:roleName=updateservice-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+# Generate bundle manifests
+gen-for-bundle:
+	operator-sdk generate kustomize manifests -q
+
+# Generate bundle and metadata, then validate generated files.
+bundle:
+	cd config/manager && kustomize edit set image controller=$(IMG)
+	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(BUNDLE_VERSION) $(BUNDLE_METADATA_OPTS)
+	operator-sdk bundle validate ./bundle
+
+# Build the bundle image.
+bundle-build:
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
```

Comment:
I do not think we have defined the bundle.Dockerfile.

Reply:
Also we need to build the bundled image in downstream as well. So somehow we need to just have one Dockerfile.

Reply:
`bundle.Dockerfile` gets generated by `operator-sdk generate bundle`, i.e. recipe `bundle`

Reply:
It can always be changed in the tmp area before it's copied to distgit.